Extensions/ExtendedMath.json

```
@@ -1,19 +1,20 @@
 {
   "author": "add_, @Jurfix",
-  "description": "Additional math functions and constants expressions:\n - Mathematical constants\n - Greatest common factor/Lowest common multiple\n - Factorial\n - Coordinates conversion for polar and isometric systems\n - ToFixed is an expression for specify a limit of decimals places\n\nConditions:\n- Is even?",
+  "description": "Expressions:\n - Mathematical constants\n - Greatest common factor/Lowest common multiple\n - Factorial\n - Coordinates conversion for polar and isometric systems\n - ToFixed: Converts a number to a string with the specified number of decimal places\n\nConditions:\n- IsEven: Check if the number is even (divisible by 2)",
   "extensionNamespace": "",
   "fullName": "Extended math support",
   "helpPath": "",
   "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLWZ1bmN0aW9uLXZhcmlhbnQiIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTIuNDIsNS4yOUMxMS4zMiw1LjE5IDEwLjM1LDYgMTAuMjUsNy4xMUwxMCwxMEgxMi44MlYxMkg5LjgyTDkuMzgsMTcuMDdDOS4xOCwxOS4yNyA3LjI0LDIwLjkgNS4wNCwyMC43QzMuNzksMjAuNTkgMi42NiwxOS45IDIsMTguODNMMy41LDE3LjMzQzMuODMsMTguMzggNC45NiwxOC45NyA2LDE4LjYzQzYuNzgsMTguMzkgNy4zMywxNy43IDcuNCwxNi44OUw3LjgyLDEySDQuODJWMTBIOEw4LjI3LDYuOTNDOC40Niw0LjczIDEwLjM5LDMuMSAxMi42LDMuMjhDMTMuODYsMy4zOSAxNSw0LjA5IDE1LjY2LDUuMTdMMTQuMTYsNi42N0MxMy45MSw1LjkgMTMuMjMsNS4zNiAxMi40Miw1LjI5TTIyLDEzLjY1TDIwLjU5LDEyLjI0TDE3Ljc2LDE1LjA3TDE0LjkzLDEyLjI0TDEzLjUsMTMuNjVMMTYuMzUsMTYuNUwxMy41LDE5LjMxTDE0LjkzLDIwLjcyTDE3Ljc2LDE3Ljg5TDIwLjU5LDIwLjcyTDIyLDE5LjMxTDE5LjE3LDE2LjVMMjIsMTMuNjVaIiAvPjwvc3ZnPg==",
   "name": "ExtendedMath",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/function-variant.svg",
   "shortDescription": "Additional math functions and constants as expressions and conditions.",
-  "version": "1.0.3",
+  "version": "1.0.5",
```

```suggestion
  "version": "1.1.0",
```

As there is a breaking change we would update to `1.1.0`

- First decimal is a major change in the conception + breaking change
- Second is a breaking change.
- Third is a simple fix that not affect the end user.

---

That's not how semver works, see the spec: https://semver.org/
A breaking change always is a major bump. A minor bump is for backwards compatible feature addition.

---

I have just pushed a new version with 2.0.0 and a section with release notes about this change. 

Big picture, I think all extensions need a new "Release notes" field added, which can be shown before applying upgrades.